homeassistant/components/axis/__init__.py

```diff
@@ -50,34 +48,10 @@ async def async_migrate_entry(hass: HomeAssistant, config_entry: ConfigEntry) ->
     """Migrate old entry."""
     _LOGGER.debug("Migrating from version %s", config_entry.version)
 
-    #  Flatten configuration but keep old data if user rollbacks HASS prior to 0.106
-    if config_entry.version == 1:
-        unique_id = config_entry.data[CONF_MAC]
-        data = {**config_entry.data, **config_entry.data[CONF_DEVICE]}
-        hass.config_entries.async_update_entry(
-            config_entry, unique_id=unique_id, data=data
-        )
-        config_entry.version = 2
-
-    # Normalise MAC address of device which also affects entity unique IDs
-    if config_entry.version == 2 and (old_unique_id := config_entry.unique_id):
-        new_unique_id = format_mac(old_unique_id)
-
-        @callback
-        def update_unique_id(entity_entry):
-            """Update unique ID of entity entry."""
-            return {
-                "new_unique_id": entity_entry.unique_id.replace(
-                    old_unique_id, new_unique_id
-                )
-            }
-
-        if old_unique_id != new_unique_id:
-            await async_migrate_entries(hass, config_entry.entry_id, update_unique_id)
-
-            hass.config_entries.async_update_entry(
-                config_entry, unique_id=new_unique_id
-            )
+    if config_entry.version != 3:
+        # Home Assistant 2023.2
+        config_entry.version = 3  # Previous version bumps where unnecessary
```

Comment:
I'd clarify or remove this comment.
```suggestion
        config_entry.version = 3
```

Reply:
Thanks!